src/Hashids.php

```diff
@@ -79,7 +100,10 @@ class Hashids implements HashidsInterface
      *
      * @return void
      */
-    public function __construct($salt = '', $minHashLength = 0, $alphabet = 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890')
+    public function __construct(
+        $salt = Hashids::DEFAULT_SALT,
```

Comment:
@roukmoute Can do but @vinkla thinks this pull request is useless and doesn't want to accept it. Do you have anything to say about it? #72